.github/workflows/build_index.yml

```diff
@@ -0,0 +1,125 @@
+name: Build index, release package
+
+on: [push]
+
+jobs:
+
+  build_release:
+
+    name: Build index, release package
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+
+    steps:
+
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.5
+
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Print tool versions
+        run: |
+          java --version
+          mvn --version
+          if [ -d ~/.m2/ ]; then find ~/.m2/; fi
+
+      # https://stackoverflow.com/a/59936630/200603
+      - name: Set up Maven to fetch from HTTPS repository
+        run: |
+          mkdir -p ~/.m2/
+          echo "<settings>
+            <mirrors>
+              <mirror>
+                <id>centralhttps</id>
+                <mirrorOf>central</mirrorOf>
+                <name>Maven central</name>
+                <url>https://repo1.maven.org/maven2/</url>
+                </mirror>
+              </mirrors>
+            </settings>" > ~/.m2/settings.xml
+
+      - name: Update APT listing
+        run: sudo apt-get -y update
+
+      - name: Install utility packages
+        run: sudo apt-get -y install pv unzip
+
+      - name: Download the latest version of allCountries.zip gazetteer file from GeoNames.org
+        run: |
+          curl -O http://download.geonames.org/export/dump/allCountries.zip
+          unzip allCountries.zip
+          rm allCountries.zip
+          mv allCountries.txt CLAVIN/
+
+      - name: Build CLAVIN
+        run: |
+          cd CLAVIN/
+          mvn compile
+          cd ../ 
+          mv ./logback.xml ./CLAVIN/target/classes/logback.xml
```

Comment:
Do you know whether GitHub has the same logging constraints as Travis? I did some Googling but couldn't find it documented anywhere.

Even if not, I think there's still a strong argument for patching since the logs get so bloated otherwise; however, I can also see the logic of avoiding patching if it's not absolutely necessary.